cli/src/main/java/org/jboss/as/cli/handlers/ShutdownHandler.java

```diff
@@ -177,11 +176,6 @@ protected void doHandle(CommandContext ctx) throws CommandLineException {
 
             if(isLocalClient) {
                 ctx.printLine("The JBoss CLI session will be closed automatically to allow the server be updated. Once the server has been restarted, you can relaunch the JBoss CLI session.", false);
-                try {
```

Comment:
The wait in the startup scripts only applies to Windows scripts, but this wait happened also on Linux - is that OK to remove it?

Reply:
@spyrkob thanks for commenting on it. It does nothing, it just waits a bit to have a better UX; you execute the command and see how it waits a bit with the message printed, that's all. It is safe to close it immediately.